ifrs17/Import/Importers.ipynb

```diff
@@ -1189,8 +1191,32 @@
                 "\n                                            if ( Enum.TryParse(interpolationMethodInput, out InterpolationMethod ipm)) ",
                 "\n                                                interpolationMethod = ipm;",
                 "\n                                            else if ( !(periodicity == (CashFlowPeriodicity)default && string.IsNullOrEmpty(interpolationMethodInput)) ) { ApplicationMessage.Log(Error.InvalidInterpolationMethod, dataNode); return null; }",
-                "\n                                        }",
-                "\n",
+                "\n                                    }",
+                "\n",
+                "\n                                    string economicBasisDriver = default;",
+                "\n                                    if(hasEconomicBasisDriver)",
+                "\n                                    {",
+                "\n                                        var economicBasisDriverInput = datarow.Field<string>(nameof(SingleDataNodeParameter.EconomicBasisDriver));",
+                "\n                                        if (!string.IsNullOrEmpty(economicBasisDriverInput))",
+                "\n                                            economicBasisDriver = economicBasisDriverInput;",
+                "\n                                        else { ApplicationMessage.Log(Error.InvalidEconomicBasisDriver, dataNode); return null; }",
```

Comment:
I think this validation is logged by the platform since in the data model you have the attribute Dimension

Reply:
not sure what the comment means @dcolleoni? was this adressed by your ValidateEconomicBasisDriver?